piIOComm.c

```diff
@@ -478,8 +476,7 @@ EGpioValue piIoComm_readSniff1A()
 
 EGpioValue piIoComm_readSniff1B()
 {
-	if (piDev_g.machine_type == REVPI_CORE ||
-	    piDev_g.machine_type == REVPI_CORE_SE) {
+	if (piDev_g.pibridge_supported && !piDev_g.only_left_pibridge) {
```

Comment:
Unnecessary `pibridge_supported` check, I don't think we ever end up here in the !pibridge_supported case.